### DISCUSSION TITLE
초보적인 질문드립니다

### DISCUSSION THREAD

**Discussion by lcg101** [tedilabs/fastcampus-devops#50] [Q&A] [answered]
혹시 AWS EC2 만드는 방법은 따로 없고 저희가 구축후에 강의 따라가면 되는건가요???

**Comment by posquit0** [comment, marked as answer]
테라폼 코드를 통해 AWS EC2 인스턴스 생성하는 내용에 대한 강의 영상이 있습니다! :)
